tools/crm_resource_runtime.c

```diff
@@ -1231,7 +1231,7 @@ max_delay_in(pe_working_set_t * data_set, GList *resources)
 }
 
 #define waiting_for_starts(d, r, h) ((d != NULL) || \
-                                    (resource_is_running_on((r), (h)) == FALSE))
+                                    (resource_is_running_on((r), (h)) == false))
```

Comment:
I prefer to use !... with booleans anyway

Otherwise logically you'd never stop e.g. ((f(r,h) == false) == true). Also I find it more readable when you can see right away a value is being negated.

Reply:
Yeah, I agree.  Fixed.